chat/src/main/java/bisq/chat/channel/ChatChannelNotificationType.java

```diff
@@ -19,22 +19,22 @@
 
 import bisq.common.proto.ProtoEnum;
 import bisq.common.util.ProtobufUtils;
+import bisq.common.util.StringUtils;
 
 public enum ChatChannelNotificationType implements ProtoEnum {
-    UNSPECIFIED,
     GLOBAL_DEFAULT, // As defined in settings
     ALL,
     MENTION,
     OFF;
 
-    private static final String protobufPrefix = "CHATCHANNELNOTIFICATIONTYPE_";
+    private static final String protobufPrefix = StringUtils.capitalizeAll(ChatChannelNotificationType.class.getSimpleName()) + "_";
```

Comment:
Can be removed as protobufPrefix is not used anymore.

Reply:
Hello, it is still used on the toProto method for all of the modified enums.